gtk/PrefsDialog.cc

```diff
@@ -1051,6 +1084,12 @@ Gtk::Widget* PrefsDialog::Impl::networkPage()
     w->set_tooltip_text(_("LPD is a tool for finding peers on your local network."));
     t->add_wide_control(row, *w);
 
+    t->add_section_title(row, _("Default Trackers"));
+    auto tv = new_text_view(TR_KEY_default_trackers, core_);
+    tv->set_tooltip_text(
+        _("a list of default trackers to be added to new public torrents (and existing ones, after a reload)"));
```

Comment:
...if we just merge these at run-time instead of saving them into newly-added torrents' announce lists, then changes made here could take effect live in libtransmission by walking through public torrents and re-merging the announce  lists from the .torrents with this

Reply:
Done.